test/avocado/run-tests

```diff
@@ -356,21 +351,8 @@ def main():
 
     opts = parser.parse_args()
 
-    regular_tests = ["check*.py"]
-    browser_tests = ["selenium-*.py"]
-
     if not opts.tests:
-        if opts.regular_tests:
-            opts.tests = regular_tests
-        if opts.selenium_tests:
-            opts.tests = browser_tests
-
-    # if we don't have a browser but are supposed to run selenium related tests, fail
-    if opts.browser == 'none':
-        for t in browser_tests:
-            if t in opts.tests:
-                sys.stderr.write("Unable to run test {0} because browser isn't set.\n".format(t))
-                return 1
+        opts.tests = ["selenium-*.py"]
 
     if len(opts.tests) == 0:
```

Comment:
This never is going to be true. Either some tests are passed or they are set up by previous line